apps/src/templates/sectionProgress/sectionProgressRedux.js

```diff
@@ -104,6 +107,12 @@ export default function sectionProgress(state=initialState, action) {
       currentView: action.viewType
     };
   }
+  if (action.type === SET_LESSON_OF_INTEREST) {
```

Comment:
Can we add a test for this redux action? Can be in a follow up PR if you prefer.

Reply:
will do in a follow up.